src/main/resources/assets/groovyscript/lang/en_us.lang

```diff
@@ -796,6 +796,40 @@ groovyscript.wiki.enderio.vat.multipliers2.value=Sets the multiplier applied to
 groovyscript.wiki.enderio.vat.baseMultiplier.value=Sets the base amount of fluid output
 groovyscript.wiki.enderio.vat.energy.value=Sets the energy cost of the recipe
 
+# EssentialCraft
+groovyscript.wiki.essentialcraft.demon_trade.title=Demon Trade
+groovyscript.wiki.essentialcraft.demon_trade.description=Adds an item that can be sold to Demons to obtain Ackronite. Note that each demon that spawns has a random item that it can accept, and will not accept any other item.
+groovyscript.wiki.essentialcraft.demon_trade.add=Adds an item or an entity that can be sold to demons.
```

Comment:
currently, this creates two periods for the `add` and `remove` on the wiki. should remove the period at the end of the lines for everything except `description`